demos/python_demos/colorization_demo/README.md

```diff
@@ -4,7 +4,9 @@ This demo demonstrates an example of using neural networks to colorize a video.
 You can use the following models with the demo:
 
 * `colorization-v2`
+* `colorization-v2-pytorch`
```

Comment:
please move this line to request with correspondent model. Now your build failed due to missed model:
```
15:49:33  No matching models: "colorization-v2-pytorch"
```